server/DatabaseHandler.py

```diff
@@ -62,6 +63,20 @@ def initCacheCollection(self):
 
     ### MISC ###
 
+    # Returns the index of the nearest beginning of a word before "currIndex"
```

Comment:
Docstring should be a triple-quoted string beneath the function name,
e.g.
def func_name(self, param):
"""
Docstring
"""